src/ProjectService.Models.Dto/Requests/RemoveImageRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using LT.DigitalOffice.ProjectService.Models.Dto.Enums;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto.Requests
+{
+    public class RemoveImageRequest
+    {
+        public Guid Id { get; set; }
+        public ImageType Image { get; set; }
```

Comment:
it will be better if you rename variable to ImageType. Here not image, here type

Reply:
done